app/Rom.php

```diff
@@ -283,6 +287,7 @@ public function setStartingEquipment(ItemCollection $items) {
 				case 'Mushroom':
 					$equipment[0x344] = 0x01;
 					$equipment[0x38C] |= 0b00100000;
+					$equipment[0x38C] |= 0b00001000;
```

Comment:
this should be merged into the line above:
```php
$equipment[0x38C] |= 0b00101000;
```

Reply:
Changed